Readme.md

```diff
@@ -152,9 +152,9 @@ description: how to configure auth
 <inline-fragment platform="ios" src="~/docs/lib/auth/setup/fragments/ios/automated.md"></inline-fragment> <inline-fragment platform="web" src="~/docs/lib/auth/setup/fragments/web/automated.md"></inline-fragment>
 ```
 
-### Tab-switchable Code Blocks
+### Tab-switchable Blocks
 
-If you want to have a code block, which has tabs to change the language/framework/etc., go ahead and utilize the amplify-code-block-switcher component. This component accepts a single prop––`tab-heading-list`––which should be a comma-separated list of headings, where each heading accounts for a child code block. Here's an example of it's usage:
+If you want to have a block, which has tabs to change the language/framework/etc., go ahead and utilize the `amplify-block-switcher` component. This component accepts a single prop––`tab-heading-list`––which should be a comma-separated list of headings, where each heading accounts for a child code block. Here's an example of it's usage:
```

Comment:
Nit: its usage.

More-than-a-nit: I don't see `tab-heading-list` in the example -- is this up to date?

Reply:
Suggestion:

> `amplify-block-switcher` allows you to organize blocks of content into tabs. This is useful for presenting a reader different instructions based upon framework (e.g. Vue.js vs. React) or language (e.g. Java vs. Kotlin). Here's an example of its usage:

Reply:
Good catch.